public/javascripts/privateSongForm.js

```diff
@@ -0,0 +1,219 @@
+var privateSongForm = function(song, type, readonly) {
+    var title = m.prop(song.title)
+    var author = m.prop(song.author)
+    var lang = m.prop(song.lang === '' ? 'english' : song.lang)
+    var lyric = song.lyric
+    var copypaste = []
+    var errors = [];
+
+    var postURL = type === 'add' ? '/user/privatesong/add' : '/user/privatesong/' + song._id + '/edit'
+
+    $(window).load(function() {
+        if (song.title === '') {
```

Comment:
is this the best way to detect that it is a new song? Is there a better way?
If you choose to go this way, make it clear why this is doing this in a comment.